docs/guides/first-shopify-abtest.mdx

```diff
@@ -0,0 +1,140 @@
+- Staying in the **Groups** panel, click on **sample_parameter** and enter a parameter name of your choice; in this case, I'm calling the parameter **image_selector**. Enter the parameter values that you want to use; in this case, I simply want to toggle between the two images that we've already uploaded to the store in the previous step
+
+![image](https://user-images.githubusercontent.com/1315028/150181235-068a1cf3-00ef-48f8-b2fa-51c0338e5e6d.png)
+
+
+### Step 3: Customize your theme
+
+After you log into your store in step 1, click on **Themes** under the **Online Store** in the left hand navigation panel. 
+Then, 
+- Click on the **Actions** drop down, then click on **Edit Code**
+- Click on **theme.liquid** file under the **Layout** section
+- Load the SDK by copying the code the below in the `head` section; use the experiment config to show the right image to your visitors
+
+```js
+<script src="https://cdn.jsdelivr.net/npm/statsig-js/build/statsig-prod-web-sdk.min.js"></script>
+```
+
+
+- Initialize the SDK and get the experiment config by copying the following in the `body` section
+
+```js
+<script>
+    // hide the page until we fetch the configuration from statsig
+    document.getElementsByTagName("BODY")[0].style.display = "none";
+    
+    //initialize the Statsig SDK using your Statsig client SDK key 
+    window.statsig.initialize("<CLIENT_SDK_KEY>").then(() => {
+        // Assuming you're on the Dawn theme and you've uploaded two images for the hero image
+        const heroImages = document.querySelectorAll(".banner > .banner__media");
+        if (heroImages.length == 2) {
+            heroImages[0].classList.remove("banner__media-half");
+            heroImages[1].classList.remove("banner__media-half");
+            
+            //get experiment config, control (default) is 0
+            const config = statsig.getExperiment("shopify_hero_image_abtest").get("image_selector", 0)
+
+            // swap the images if the user is assigned to treatment  
+            if (config == 1){
+              const primaryHTML = heroImages[0].innerHTML;
+              const primaryOuter = heroImages[0].outerHTML;
+              
+              heroImages[0].innerHTML = heroImages[1].innerHTML;
+              heroImages[0].outerHTML = heroImages[1].outerHTML;
+              
+              heroImages[1].innerHTML = primaryHTML;
+              heroImages[1].outerHTML = primaryOuter;
+            }
+        }
+        // show the page
+        document.getElementsByTagName("BODY")[0].style.display = "block";                                         
+    });
+</script>
+```
+
+- Log the key events that you want to track as metrics (along with any metadata) by copying the code below into the pages where customers take key actions
+
+```js
+statsig.logEvent('purchase_event', 'SKU_12345', {'price': '9.99', 'item_name': 'diet_coke_48_pack'});
```

Comment:
This is from other docs, but we should update the value to be the product category - thats our current suggestion, anyway